src/microbe_stage/editor/CellEditorComponent.GUI.cs

```diff
@@ -285,6 +285,31 @@ private void UpdateOrganelleUnlockTooltips()
         }
     }
 
+    private void UpdateOrganelleLAWKSettings()
+    {
+        // Don't use placeablePartSelectionElements as the thermoplast isn't placeable yet but is LAWK-dependent
+        foreach (var organelle in allPartSelectionElements.Keys)
+        {
+            var control = allPartSelectionElements[organelle];
```

Comment:
This kind of loop isn't very uniform with style elsewhere.
Could you reformat this and the next few loops to be like:
```
foreach (var entry in allPartSelectionElements)
...
```
or `foreach (var item in allPartSelectionElements.Items)`?